src/main/java/org/mskcc/cbio/oncokb/config/application/ApplicationProperties.java

```diff
@@ -3,6 +3,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.mskcc.cbio.oncokb.domain.enumeration.ProjectProfile;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.mskcc.oncokb.meta.model.application.RedisProperties;
```

Comment:
I guess this reference is no longer needed?

Reply:
Its the type for the **redis** field.